File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        hashmap = {}

        for i in range(len(nums)) :

            x = target - nums[i]
            if x in hashmap:
                
                return [hashmap[x], i]
            
            hashmap[nums[i]] = i

File: 1-two-sum/two-sum.go
func twoSum(nums []int, target int) []int {
    
    for i:= 0; i < len(nums); i++{
        for j:= i + 1; j < len(nums); j++{
            if nums[i] + nums[j] == target{
                ans := []int{i , j}
                return ans
            }
        }
    }
    return nums
}
